README.md

```diff
@@ -1,8 +1,6 @@
 # dcos-images
 
-The reference for DC/OS images and how to build them.  
-The goal of this repo is to standardize, across all DC/OS users and developers, the usage and build process of OS cloud
-images to run DC/OS.
+Repository is used to qualify DC/OS releases against CentOS and RHEL operating systems.
```

Comment:
You are also using it to qualify FlatCar Linux for the time being right?

Reply:
We are in the process of figuring out our minimal os strategy.